packages/styling/src/classNames/colorClassNames.ts

```diff
@@ -3,6 +3,7 @@ import { css, CSSProperties } from 'glamor';
 import { defaultColorStyles } from '../styles/colorStyles';
 
 export interface IColorClassNames {
+  [index: string]: string;
```

Comment:
It gets accessed by an index below:
```typescript
const style: CSSProperties = { [cssProperty]: getTheme().colors[colorName] };
```
Which isn't typesafe since colorName is a string, so colors[colorName] is implicitly of type any. But, it does have the side effect of letting you reference colors.foo, which isn't ideal. I'll see if we can do something else.